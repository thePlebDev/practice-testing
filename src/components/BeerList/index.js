import React from 'react';

const BeerList =()=>{

  return(
    <ul/>
  )
}

export default BeerList
