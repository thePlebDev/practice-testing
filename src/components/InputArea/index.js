import React from 'react';


const InputArea = ()=>{


  return(
    <input/>
  )
}

export default InputArea
