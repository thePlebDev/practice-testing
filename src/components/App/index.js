import React from 'react';

import BeerList from '../BeerList';

const App = ()=>{

  return(
    <div>
      <BeerList />
    </div>
  )
}

export default App;
