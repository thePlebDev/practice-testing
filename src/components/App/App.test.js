import React from 'react';


import { shallow } from 'enzyme';

import App from './index.js';
import BeerList from '../BeerList'
import InputArea from '../InputArea'


describe('BeerListContainer',()=>{
  it('should render InputArea and BeerList',() =>{
    const wrapper = shallow(<App />);
    expect(wrapper.containsAllMatchingElements([
      <InputArea/>,
      <App/>
    ])).to.equal(true)
  })
})
