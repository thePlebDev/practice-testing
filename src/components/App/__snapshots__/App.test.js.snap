// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshots 1`] = `
<div>
  <h1>
    the APP that smiles back
  </h1>
</div>
`;
